modules/tinymce/src/plugins/help/main/ts/ui/KeyboardNavTab.ts

```diff
@@ -3,68 +3,91 @@ import { Dialog } from 'tinymce/core/api/ui/Ui';
 // TODO: When we translate this, we should pull the HTML out into a HTML file, the way TBIO does it.
 // That requires webpack and rollup changes though, so inlining it for now.
 /* eslint-disable max-len */
-const description = `<h1>Editor UI keyboard navigation</h1>
+const description = `<h1>Activate keyboard navigation</h1>
 
-<h2>Activating keyboard navigation</h2>
+<dl>
+  <dt>Activate the Menu bar</dt>
```

Comment:
"Activate" denotes putting the element in an `active` state. We should use "Focus" instead since the elements are being put in `focus` state.
```suggestion
  <dt>Focus the Menu bar</dt>
```